### DISCUSSION TITLE
UX improvements

### DISCUSSION THREAD

**Discussion by APraxx** [SamTV12345/PodFetch#155] [General]
Some things that i think would improve the UX

 - playback status in Episode list (at least already played)

more minor:
- Notification handling
  - dismiss when not in focus anymore
  - clear all

- Some Feedback/Toast for
  - while adding/importing
  - accepted invite (it just stays the same)

ps: not affiliated with the reddit comment

**Comment by SamTV12345** [comment]
Good ideas. I'll try to complete this tomorrow.

**Reply by SamTV12345** [reply]
Could you draw something or give an example of how that progress bar would look like in the episode list?

**Reply by APraxx** [reply]
i dont know if a progress bar is really needed or the right element? maybe just graying out already finished episodes is enough.

i see if i can do a mockup for both

**Reply by APraxx** [reply]
![grafik](https://github.com/SamTV12345/PodFetch/assets/6854060/746e63c7-e628-47f6-9d29-80ca9ba26aa7)

how is this? 
turned the opacity to .8 of the played one. hope it doesn't highlight it too much. the stop icon is maybe misplaced as it could be misinterpreted. 
the progress is self explanatory but don't know if we need this much or maybe change it to percentage

**Reply by APraxx** [reply]
another version, this time in text
![grafik](https://github.com/SamTV12345/PodFetch/assets/6854060/f74e772d-ab51-4485-8959-4f7c853ec919)

did also a pause icon for the currently playing but that is just for the mockup if too difficult

**Reply by SamTV12345** [reply]
Shouldn't be too hard to implement.  Thanks for the draft.

**Reply by SamTV12345** [reply]
@jchue @APraxx  I am currently implementing the current watch time of an episode in the podcast detail pages. Now that the UI changed a lot. Where should we put the information how much a user listened of a podcast? Could someone create a wireframe for that please? I could do it but I'm always so clueless when it comes to user experience 😄

**Reply by APraxx** [reply]
from my point of view the mockups are still the way i'd align it

**Reply by Deigue** [reply]
To add to this , a circular green check icon (image lookup: green check svg) , along with the timestamp of when it was watched would be good. Along with what was suggested above, fading/graying the background of the podcast episode is a good/common means of showing that it has already been gone through.

**Reply by jchue** [reply]
Let me see if I can incorporate this into the latest wireframes.
